examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -61,6 +64,13 @@ NamedPipeCommands sChipNamedPipeCommands;
 AllClustersCommandDelegate sAllClustersCommandDelegate;
 chip::app::Clusters::WindowCovering::WindowCoveringManager sWindowCoveringManager;
 
+chip::app::Clusters::ResourceMonitoring::HepaFilterMonitoringInstance
+    gHepafilterInstance(0x1, chip::app::Clusters::HepaFilterMonitoring::Id, 3,
```

Comment:
For the feature map, the 3 is a bit inscrutable.  An actual BitSet with the right named bits would be clearer...

Reply:
done